iquip/apps/builder.py

```diff
@@ -352,7 +356,7 @@ def onSubmitted(self, rid: int):
         
         TODO(BECATRUE): It will be developed in Log Viewer project.
         """
-        print(f"RID: {rid}")
+        logger.info("RID: %d", rid)
```

Comment:
How about updating the method docstring as follows:
```
Sends the rid to the logger after submitted.
```

This is just an example, and if you think 'prints' is still ok, never mind!

Reply:
Also, please remove the above TODO comment because you implemented it instead of me.